proto/dymensionxyz/dymension/eibc/tx.proto

```diff
@@ -22,7 +22,7 @@ message MsgFulfillOrder {
     string fulfiller_address = 1;
     // order_id is the unique identifier of the order to be fulfilled.
     string order_id = 2;
-    // expected_fee is the nominal fee set in the order. 
+    // expected_fee is the nominal fee set in the order. Fulfiller will generally make less profit (after deducting bridge fee)
     string expected_fee = 3;
```

Comment:
fwiw, the `expected_fee` is here to avoid "front running" the fulfill tx, with update_fee tx
so it's kind of `demand_order_fee`

Reply:
Yeah IK 👍